Testing/PythonTests/padfail.py

```diff
@@ -1,12 +1,12 @@
 import os
 import sys
-from shapeworks import *
+import shapeworks as sw
 
 def padfailTest1():
-  img = Image(os.environ["DATA"] + "/1x2x2.nrrd")
+  img = sw.Image(os.environ["DATA"] + "/1x2x2.nrrd")
   img.pad()
 
-  compareImg = Image(os.environ["DATA"] + "/padfail.nrrd")
+  compareImg = sw.Imageage(os.environ["DATA"] + "/padfail.nrrd")
```

Comment:
This fails for the wrong reasons @archanasri

Reply:
Looks like it is fixed.